envoy/http/async_client.h

```diff
@@ -3,18 +3,24 @@
 #include <chrono>
 #include <memory>
 
+#include "envoy/config/core/v3/base.pb.h"
 #include "envoy/config/route/v3/route_components.pb.h"
 #include "envoy/event/dispatcher.h"
 #include "envoy/http/message.h"
 #include "envoy/stream_info/stream_info.h"
 #include "envoy/tracing/http_tracer.h"
 
 #include "source/common/protobuf/protobuf.h"
+#include "source/common/protobuf/utility.h"
 
 #include "absl/types/optional.h"
 
 namespace Envoy {
 namespace Http {
+namespace {
+static constexpr uint32_t RetryInitialDelayMilliseconds = 1000;
```

Comment:
nit: constexpr implies static
```suggestion
constexpr uint32_t RetryInitialDelayMilliseconds = 1000;
```